index.js

```diff
@@ -20,10 +20,21 @@ const concatArray = (obj, src) => {
   return undefined
 }
 
+const isString = x => typeof x === 'string'
+
+const resolvePresets = (presets, cwd) =>
+  presets.map(
+    preset => (isString(preset) ? resolvePreset(preset, cwd) : preset)
+  )
+
+const resolvePlugins = (plugins, cwd) =>
+  plugins.map(
+    plugin => (isString(plugin) ? resolvePlugin(plugin, cwd) : plugin)
+  )
+
```

Comment:
Duplicate.
I like the abstraction like `createResolveList`

```js
const createResolveList = fn => (list, cwd) =>
  list.map(x => (typeof x === 'string' ? fn(x, cwd) : x))

const resolvePresets = createResolveList(resolvePreset)
const resolvePlugins = createResolveList(resolvePlugin)
```

Reply:
I rebased against master and applied your suggestion